tests/unit/util/newconfig/ConfigValueTests.cpp

```diff
@@ -105,14 +214,14 @@ TEST(ConfigValue, OneOfConstraint)
     );
 }
 
-TEST(ConfigValue, IpConstraint)
+TEST_F(ConstraintTest, IpConstraint)
 {
     auto ip = ConfigValue{ConfigType::String}.defaultValue("127.0.0.1").withConstraint(gValidateIp);
     EXPECT_FALSE(ip.setValue("http://127.0.0.1").has_value());
     EXPECT_FALSE(ip.setValue("http://127.0.0.1.com").has_value());
     auto const err = ip.setValue("123.44");
     EXPECT_TRUE(err.has_value());
-    EXPECT_EQ(err->error, "Ip is not a valid ip address");
+    EXPECT_EQ(err->error, "Unknown_key Ip is not a valid ip address");
```

Comment:
nit: The final error message reads really strangely. `[something] Ip is not a valid ip address`. Maybe `The value of [something] is not a valid IP address` or similar is better?

Reply:
I think it is out of the PR's scope, because such style of errors is currently used across 'newconfig` library, so changing it only here would make it worse. I will create a task for refactoring.

Reply:
Created #1886.